.github/workflows/ci.yml

```diff
@@ -22,7 +22,25 @@ jobs:
       # and hyphens.
       run: echo "TAG=branch-${GITHUB_REF_NAME//[^A-Za-z0-9._-]/-}" | tee -a $GITHUB_ENV
 
-    - run: ./devel/build $TAG
+    - uses: docker/setup-qemu-action@v2
+
+      # Adapted from https://github.com/actions/runner-images/issues/6180#issuecomment-1236674387
+    - name: Install newer version of skopeo
+      run: |
+        sudo apt purge golang-github-containers-common skopeo
+        sudo apt autoremove --purge
+        REPO_URL="https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable"
+        source /etc/os-release
+        sudo sh -c "echo 'deb ${REPO_URL}/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list"
+        sudo wget -qnv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/x${NAME}_${VERSION_ID}/Release.key -O Release.key
+        sudo apt-key add Release.key
+        sudo apt-get update
+        sudo apt-get install skopeo
+        skopeo --version
+
+    - run: ./devel/start-localhost-registry
+
```

Comment:
This should have a teardown script paired with it.

Reply:
Added in acbde816baf00f147e1d455e4691257684e5e21f:

https://github.com/nextstrain/docker-base/blob/acbde816baf00f147e1d455e4691257684e5e21f/devel/stop-localhost-registry#L3-L9